au3/libraries/lib-dynamic-range-processor/DynamicRangeProcessorTypes.h

```diff
@@ -53,14 +45,17 @@ struct Unbypassed
 
 static constexpr auto compressorMeterUpdatePeriodMs = 1000 / 30;
```

Comment:
It is better to use a type here, because it is not clear if the integer division was intended or not, considering that the variables below are doubles.